rest/model/project.go

```diff
@@ -88,13 +91,18 @@ func (t *APITriggerDefinition) BuildFromService(triggerDef model.TriggerDefiniti
 }
 
 type APIPatchTriggerDefinition struct {
-	Alias                  *string            `json:"alias"`
-	ChildProjectId         *string            `json:"child_project_id"`
-	ChildProjectIdentifier *string            `json:"child_project_identifier"`
-	TaskSpecifiers         []APITaskSpecifier `json:"task_specifiers"`
-	Status                 *string            `json:"status,omitempty"`
-	ParentAsModule         *string            `json:"parent_as_module,omitempty"`
-	VariantsTasks          []VariantTask      `json:"variants_tasks,omitempty"`
+	// Alias to run in the downstream project.
+	Alias *string `json:"alias"`
+	// ID of the downstream project.
+	ChildProjectId *string `json:"child_project_id"`
+	// Identifier of the downstream project.
+	ChildProjectIdentifier *string `json:"child_project_identifier"`
+	// List of task specifiers.
+	TaskSpecifiers []APITaskSpecifier `json:"task_specifiers"`
+	// Status for the parent patch to conditionally kick off the child patch.
+	Status         *string       `json:"status,omitempty"`
+	ParentAsModule *string       `json:"parent_as_module,omitempty"`
```

Comment:
I just wrote some better docs for this in https://github.com/evergreen-ci/evergreen/pull/7390 -- this means the module name corresponding to the upstream project in the downstream project's yaml (this needs to be included if you want the child patch to test with the upstream project's changes)

Reply:
Thanks, updated!